Gemfile

```diff
@@ -28,6 +28,7 @@ group :development, :test do
   gem "capybara"
   gem "factory_bot_rails"
   gem "govuk-lint", "~> 3"
+  gem 'pry'
```

Comment:
For future PRs, could you please split them in atomic commits? In this case, adding `pry` fits better in its own commit, along with an explanation of why it is better than byebug.